Account/FxADevice.swift

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+
+public struct FxADevice {
+    let name: String
+    let id: String?
+    let type: String?
+    let isCurrentDevice: Bool?
```

Comment:
I wonder if we can make this non-optional and default it to `false` if we never receive anything. Might help with the code a few layers up instead of having to conditional unwrap it later.